Wireframe/css/style.css

```diff
@@ -0,0 +1,103 @@
+body {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    text-align: center;
+    background: linear-gradient(#D7CCC8, #908f89);
+    color: #FAEBD7;
+    margin: 0;
+}
+
+#logo {
+    width: 10rem;
+    vertical-align: middle;
+}
+
+img {
+    width: 10rem;
+    border-radius: 1rem;
+}
+
+header {
+    color: #780000;
+    background-color: #FAEBD7;
+    background: radial-gradient( #DAD7CD 50%, #908f89);
+    margin: 0 auto;
+    text-shadow:  0 0 1px white;
+    box-shadow: 0 0 15px #780000;
+    padding: 10px;
+}
+
+main {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 3vw auto;
+}
+
+h3 {
+    text-shadow: 0 0 5px black;
+}
+
+article {
+    border: 4px solid #780000;
+    border-radius: 30px;
+    padding: 2rem;
+    text-shadow: 0 0 2px black;
+    box-shadow: 0 0 10px black;
+}
+
+article:first-child {
+    background: radial-gradient(#34779b, #003049);
+    width: 90vw;
+    margin: 3vw auto 2vw;
+}
+
+#second, #third {
+    width: 40vw;
+    margin: 3vw auto;
+    background: radial-gradient( #003049, #34779b);
+}
+
+button {
+    background-color: #DAD7CD;
+    height: 3rem;
+    width: 12rem;
+    border-radius: 10px;
+    box-shadow: 0 0 2px black;
+}
+
+.btn {
+    display: inline-block;
+    padding: 10px 20px;
+    background-color: #DAD7CD;
+    color: #003049;
+    text-decoration: none;
+    border-radius: 5px;
+    border: none;
+    cursor: pointer;
+    transition: 0.3s;
+    text-shadow:  0 0 1px #dcdcdc;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    font-size: 1rem;
+    font-weight: bold;
+    box-shadow: 0 0 10px black;
+}
+  
+.btn:hover {
+    background-color: #908f89;
+}
+
+footer {
+    color: #003049;
+    background: linear-gradient(0deg, #908f89, #DAD7CD);
+    margin: 0;
+    position: relative;
+    bottom: 0;
```

Comment:
Does this meet the acceptance criteria for this work? (Even if not, it looks great though!)

Reply:
You're right.
Actually, I intended for the footer to be fixed at the bottom of the viewport, but I accidentally made it sticky to the bottom of the entire page instead.
I’ll make the necessary adjustments to ensure it meets the requirements.